code/modules/clothing/modular/modular.dm

```diff
@@ -0,0 +1,217 @@
+/**
+	Modular armor
+*/
+/obj/item/clothing/suit/modular
+	name = "Jaeger XM-02 combat exoskeleton"
+	desc = "Designed to mount a variety of modular armor components and support systems. It comes installed with light-plating and a shoulder lamp. Mount armor pieces to it by clicking on the frame with the components"
+	icon = 'icons/mob/modular/modular_armor.dmi'
+	icon_state = "underarmor_icon"
+	item_state = "underarmor"
+	flags_armor_protection = CHEST|GROIN|ARMS|LEGS
+	/// What is allowed to be equipped in suit storage
+	allowed = list(
+		/obj/item/weapon/gun/,
+		/obj/item/tank/emergency_oxygen,
+		/obj/item/storage/bible,
+		/obj/item/storage/belt/sparepouch,
+		/obj/item/storage/large_holster/machete,
+		/obj/item/weapon/claymore,
+		/obj/item/storage/belt/gun
+	)
```

Comment:
This whole list was just stolen elsewhere, it could use a look over

Reply:
Looks good now.